### PR TITLE
[15.0] [MIG] product_form_account_move_line_link

### PR DESCRIPTION
**Pull request by JasminSForgeFlow**
Standard Migration

### CODE REVIEW

File: product_form_account_move_line_link/__manifest__.py [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+{
+    "name": "Product From Account Move Line Link",
+    "summary": """
+        Adds a button on product forms to access Journal Items""",
+    "version": "15.0.1.0.0",
+    "author": "ForgeFlow, Odoo Community Association (OCA)",
+    "website": "https://github.com/OCA/account-invoicing",
+    "category": "Account",
+    "license": "AGPL-3",
+    "depends": ["account", "product"],
```
Comment (MiquelRForgeFlow) [inline product_form_account_move_line_link/__manifest__.py]:
```suggestion
    "depends": ["account"],
```

**Review by MiquelRForgeFlow — APPROVED** [review]

File: product_form_account_move_line_link/__manifest__.py (hunk 1 above)
Comment (JasminSForgeFlow) [reply product_form_account_move_line_link/__manifest__.py]: done, thanks

**Review by JaumeBforgeFlow — APPROVED** [review]
LGTM!

**Review by AaronHForgeFlow — APPROVED** [review]
code review

**Review by JuanyDForgeflow — APPROVED** [review]
LGTM Functional + Code

**Comment by MiquelRForgeFlow** [comment]
@JasminSForgeFlow please rebase to solve merging error

**Comment by JasminSForgeFlow** [comment]
> @JasminSForgeFlow please rebase to solve merging error

Done, Thanks

**Pull request merged**